README.md

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
 
 # GSConnect
 
-[<img src="https://raw.githubusercontent.com/andyholmes/gnome-shell-extensions-badge/master/get-it-on-ego.svg?sanitize=true" alt="Get it on GNOME Extensions" height="100" align="middle">][ego] [<img alt="Available in the Chrome Web Store" src="https://storage.googleapis.com/chrome-gcs-uploader.appspot.com/image/WlD8wC6g8khYWPJUsQceQkhXSlv1/UV4C4ybeBTsZt43U4xis.png" align="middle" hspace="12"/>][chrome] [<img src="https://ffp4g1ylyit3jdyti1hqcvtb-wpengine.netdna-ssl.com/addons/files/2015/11/get-the-addon.png" alt="Get the Add-On" align="middle">][firefox]
+[<img src="https://raw.githubusercontent.com/andyholmes/gnome-shell-extensions-badge/master/get-it-on-ego.svg?sanitize=true" alt="Get it on GNOME Extensions" height="100" align="middle">][ego] [<img alt="Available in the Chrome Web Store" src="data/images/chrome-badge.png" align="middle" hspace="12"/>][chrome] [<img src="data/images/firefox-badge.png" alt="Get the Add-On" align="middle">][firefox]
```

Comment:
Well, lovely. Mozilla's [blog post](https://extensionworkshop.com/documentation/publish/promoting-your-extension/) where they provide the badges contains no license information whatsoever. Nor does the source of the SVG file for the artwork.

The _website_ has a blanket Creative Commons Attribution Share-Alike 3.0 Unported license attached to it, "unless otherwise indicated" which I guess it's not. So looks like it's CC-BY-SA-3.0 for the badge, too.

Reply:
> If we're going to do this, I think the README should still use _full_ links, in case it ends up somewhere else where the relative links aren't valid.
> 
> Anyone checked on the license for `firefox-badge.png` and `chrome-badge.png`? Do we have them right in the REUSE metadata?
> 
> [Looks like](https://github.com/GSConnect/gnome-shell-extension-gsconnect/blob/main/.reuse/dep5) it's currently set to GPL-2.0-or-later for all images in the directory, which I'm suspicious of because Mozilla would probably use either MPL or a Creative Commons license.

I used local link in order to not break links in the case of a fork for exemple, but both are fine IMO.

Reply:
> Well, lovely. Mozilla's [blog post](https://extensionworkshop.com/documentation/publish/promoting-your-extension/) where they provide the badges contains no license information whatsoever. Nor does the source of the SVG file for the artwork.
> 
> The _website_ has a blanket Creative Commons Attribution Share-Alike 3.0 Unported license attached to it, "unless otherwise indicated" which I guess it's not. So looks like it's CC-BY-SA-3.0 for the badge, too.

I will fix the REUSE stuff for the Mozilla badge.

Reply:
The same website footer that provides the license says, 
> Portions of this content are ©1998–2024 by individual mozilla.org contributors. 

So I guess basically that: 
```text
File: data/images/firefox-badge.png
Copyright: 1998–2024 by individual mozilla.org contributors <>
License: CC-BY-SA-3.0
```

You'll also need to `reuse download` the license, if it hasn't already been used in the project.